src/rev_ai/models/__init__.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 """Models"""
 
-from models.job import Job
-from models.job_options import JobSubmitOptions
-from models.account import Account
-from models.transcript import Transcript
\ No newline at end of file
+from .job import Job
```

Comment:
@JDongian , is it best practice to use explicit relative imports?
https://stackoverflow.com/questions/22511792/python-from-dotpackage-import-syntax

Reply:
it's standard practice.